### ISSUE TITLE
ClassNotFoundError in v6.8.15

### ISSUE THREAD

**Issue by ben-manes** [cbeust/testng#593]
The latest release has a wrong import, requiring a dependency on Guice.

Since both TestNG and Guice embed Guava-like classes, personally I use a checkstyle configurations to fail a build if detected. I recommend doing the same and, preferably, choosing a unique name for your utility classes. 

```
Caused by: java.lang.NoClassDefFoundError: com/google/inject/internal/Lists
	at org.testng.internal.annotations.JDK15TagFactory.findInheritedStringArray(JDK15TagFactory.java:500)
	at org.testng.internal.annotations.JDK15TagFactory.createTestTag(JDK15TagFactory.java:388)
	at org.testng.internal.annotations.JDK15TagFactory.createTag(JDK15TagFactory.java:73)
	at org.testng.internal.annotations.JDK15AnnotationFinder.findAnnotation(JDK15AnnotationFinder.java:177)
	at org.testng.internal.annotations.JDK15AnnotationFinder.findAnnotation(JDK15AnnotationFinder.java:149)
	at org.testng.TestNG.createCommandLineSuitesForClasses(TestNG.java:567)
	at org.testng.TestNG.initializeCommandLineSuites(TestNG.java:810)
	at org.testng.TestNG.run(TestNG.java:1033)
	at org.gradle.api.internal.tasks.testing.testng.TestNGTestClassProcessor.stop(TestNGTestClassProcessor.java:115)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.stop(SuiteTestClassProcessor.java:57)
	... 21 more
Caused by: java.lang.ClassNotFoundException: com.google.inject.internal.Lists
	at java.net.URLClassLoader$1.run(URLClassLoader.java:372)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:361)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:360)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:308)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 31 more
```

**Comment by VladRassokhin** [comment]
Fixed in 8b61830cd8e

**Comment by ben-manes** [comment]
Confirmed fixed in v6.8.17.
